blockchain/vm/eips.go

```diff
@@ -119,6 +122,43 @@ func enableIstanbulComputationCostModification(jt *JumpTable) {
 	jt[SAR].computationCost = params.SarComputationCostIstanbul
 }
 
+// enable1153 applies EIP-1153 "Transient Storage"
+// - Adds TLOAD that reads from transient storage
+// - Adds TSTORE that writes to transient storage
+func enable1153(jt *JumpTable) {
+	jt[TLOAD] = &operation{
+		execute:         opTload,
+		constantGas:     params.WarmStorageReadCostEIP2929,
+		minStack:        minStack(1, 1),
+		maxStack:        maxStack(1, 1),
+		computationCost: params.TloadComputationCost,
+	}
+
+	jt[TSTORE] = &operation{
+		execute:         opTstore,
+		constantGas:     params.WarmStorageReadCostEIP2929,
+		minStack:        minStack(2, 0),
+		maxStack:        maxStack(2, 0),
+		computationCost: params.TstoreComputationCost,
+	}
+}
+
+// opTload implements TLOAD opcode
+func opTload(pc *uint64, evm *EVM, scope *ScopeContext) ([]byte, error) {
+	loc := scope.Stack.Peek()
+	val := evm.StateDB.GetTransientState(scope.Contract.Address(), common.BigToHash(loc))
+	loc.SetBytes(val.Bytes())
+	return nil, nil
+}
+
+// opTstore implements TSTORE opcode
+func opTstore(pc *uint64, evm *EVM, scope *ScopeContext) ([]byte, error) {
```

Comment:
```suggestion
func opTstore(pc *uint64, evm *EVM, scope *ScopeContext) ([]byte, error) {
	if evm.interpreter.readOnly {
		return nil, ErrWriteProtection
	}
```

Reply:
I missed this change has applied at opMcopy. I'll apply the rest of the code. Thank you.
```
	if interpreter.readOnly {
		return nil, ErrWriteProtection
	}
```

Reply:
Applied at https://github.com/klaytn/klaytn/pull/1988/commits/9fe8b33956ae04106cf25a226c1533ef032d0627. I'll apply the rest of the code in later PR!

Reply:
Applied at https://github.com/klaytn/klaytn/pull/1988/commits/929848de661c8499b9368ab7f9144dc02d104e35. I'll apply the rest of the code (opSstore, opCreate, opCreate2, opCall, opSelfDestruct, makeLog) in later PR.